addons/website_event/models/event_event.py

```diff
@@ -338,13 +330,15 @@ def _get_website_menu_entries(self):
           * menu_type: type of menu entry (used in inheriting modules to ease
             menu management; not used in this module in 13.3 due to technical
             limitations);
+          * parent_menu_type: menu_type of already created menu entry (used for
+            making submenu of existing menu entry)
         """
         self.ensure_one()
         return [
-            (_('Introduction'), False, 'website_event.template_intro', 1, 'introduction'),
-            (_('Location'), False, 'website_event.template_location', 50, 'location'),
-            (_('Info'), '/event/%s/register' % self.env['ir.http']._slug(self), False, 100, 'register'),
-            (_('Community'), '/event/%s/community' % self.env['ir.http']._slug(self), False, 80, 'community'),
+            (_('Community'), '#', False, 80, 'community', False),
+            (_('Home'), False, 'website_event.template_home', 1, 'home', False),
+            (_('Practical'), '/event/%s/register' % self.env['ir.http']._slug(self), False, 100, 'register', False),
+            (_('Rooms'), '/event/%s/community' % self.env['ir.http']._slug(self), False, 80, 'community', 'community'),
```

Comment:
1st point: :+1: 
2nd point: From what I know, it was done because community menu is used is 2 modules
`website_event_meet` and `website_event_track_quiz` and both of them are not inter dependent :thinking: